game/game.js

```diff
@@ -0,0 +1,92 @@
+// @ts-check
+
+const fs = require("fs");
+
+const Account = require("./account");
+const Password = require("./password");
+const db = require("./database");
+
+class Game {
+    /**
+     * @param {string} [shaderDirectory="./public/shaders"]
+     */
+    constructor(shaderDirectory = "./public/shaders") {
+        /**
+         * @type {string}
+         */
+        this.defaultVertexShaderCode = fs.readFileSync(
+            `${shaderDirectory}/default/shader.vert.glsl`,
+            { encoding: "utf8" },
+        );
+
+        /**
+         * @type {string}
+         */
+        this.defaultFragmentShaderCode = fs.readFileSync(
+            `${shaderDirectory}/default/shader.frag.glsl`,
+            { encoding: "utf8" },
+        );
+
+        /**
+         * @type {Array<string>} base64 images
+         */
+        this.images = require("./images.json");
+
+        /**
+         * @type {Array<Account>}
+         */
+        this.accounts = [];
+
+        this.load();
```

Comment:
yes, is async but there's definitely always some time between this call and first use so is cool don't worry about it probably